libs/extensions/include/TTAlpsHistogramsFiller.hpp

```diff
@@ -19,12 +21,22 @@ class TTAlpsHistogramsFiller {
   void FillTriggerVariablesPerTriggerSet(const std::shared_ptr<Event> event, std::string ttbarCategory = "");
 
   void FillHistograms1D(const std::shared_ptr<Event> event);
+  void FillDefaultHistograms1D(const std::shared_ptr<Event> event);
+  void FillLeadingPt(const std::shared_ptr<Event> event, std::string histName, std::vector<std::string> variableLocation);
+  void FillSubLeadingPt(const std::shared_ptr<Event> event, std::string histName, std::vector<std::string> variableLocation);
+
+  void FillCutFlowHist(const std::shared_ptr<CutFlowManager> cutFlowManager);
+
+  void FillTTAlpsHists(const std::shared_ptr<Event> event);
```

Comment:
Yeah, these names are not super consistent ;) Since it's a HistogramFiller, perhaps we can drop all "hists" and "histograms" from method names, as you did for Pt?